src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,470 @@
+  display: flex;
+  flex-direction: column;
+  grid-column: 1 / -1;
+  grid-row: 2 / 3;
+  min-width: 0;
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    grid-column: 1 / 2;
+    grid-row: 2 / 4;
+    box-shadow: var(--pf-c-wizard__steps--lg--BoxShadow);
+  }
+
+  // Mobile expanded state
+  &.pf-m-expanded {
+    grid-row: 2 / 4;
+
+    .pf-c-wizard__nav {
+      display: block;
+      visibility: visible;
+    }
+
+    .pf-c-wizard__toggle-icon {
+      transform: rotate(180deg);
+    }
+  }
+}
+
+.pf-c-wizard__toggle {
+  position: relative;
+  z-index: var(--pf-c-wizard__toggle--ZIndex);
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: var(--pf-c-wizard__toggle--PaddingTop) var(--pf-c-wizard__toggle--PaddingRight) var(--pf-c-wizard__toggle--PaddingBottom) var(--pf-c-wizard__toggle--PaddingLeft);
+  border: 0;
+  box-shadow: var(--pf-c-wizard__toggle--BoxShadow);
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    display: none;
+    visibility: hidden;
+  }
+}
+
+.pf-c-wizard__toggle-list {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  margin-right: var(--pf-c-wizard__toggle-list--MarginRight);
+  margin-bottom: var(--pf-c-wizard__toggle-list--MarginBottom);
+  list-style: none;
+}
+
+.pf-c-wizard__toggle-list-item {
+  margin-bottom: var(--pf-c-wizard__toggle-list-item--MarginBottom);
+  text-align: left;
+  word-break: break-word;
+
+  &:not(:first-child) {
+    font-size: var(--pf-c-wizard__toggle-list-item--not-first-child--FontSize);
+  }
+
+  &:not(:last-child) {
+    margin-right: var(--pf-c-wizard__toggle-list-item--not-last-child--MarginRight);
+  }
+}
+
+.pf-c-wizard__toggle-divider {
+  margin-left: var(--pf-c-wizard__toggle-divider--MarginLeft);
+}
+
+.pf-c-wizard__toggle-icon {
+  margin-top: var(--pf-c-wizard__toggle-icon--MarginTop);
+}
+
+.pf-c-wizard__nav {
+  display: none;
+  flex-grow: 1;
+  padding-top: var(--pf-c-wizard__nav--PaddingTop);
+  padding-right: var(--pf-c-wizard__nav--PaddingRight);
+  padding-left: var(--pf-c-wizard__nav--PaddingLeft);
+  overflow: auto;
+  -webkit-overflow-scrolling: touch;
+  visibility: hidden;
+  box-shadow: var(--pf-c-wizard__nav--BoxShadow);
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    position: relative;
+    display: block;
+    visibility: visible;
+  }
+
+  // Fix FF bug https://github.com/w3c/csswg-drafts/issues/129
+  &::after {
+    display: block;
+    height: var(--pf-c-wizard__nav--PaddingBottom);
+    content: "";
+  }
+}
+
+.pf-c-wizard__nav-list {
+  list-style: none;
+  counter-reset: wizard-nav-count;
+
+  // Nested sub menus
+  & & {
+    margin-top: var(--pf-c-wizard__nav--nested--MarginTop);
+    margin-left: var(--pf-c-wizard__nav--nested--MarginLeft);
+
+    // Reset styles for nested nav links
+    .pf-c-wizard__nav-link {
+      padding-left: 0;
```

Comment:
do you need to reset padding here? I don't see it being set anywhere else.

Reply:
nope! that's leftover from a previous iteration.